src/ShaderGen/ShaderMethodVisitor.cs

```diff
@@ -205,6 +205,12 @@ public override string VisitInvocationExpression(InvocationExpressionSyntax node
                 SymbolInfo symbolInfo = GetModel(node).GetSymbolInfo(ins);
                 string type = symbolInfo.Symbol.ContainingType.ToDisplayString();
                 string method = symbolInfo.Symbol.Name;
+
+                if (type == "ShaderGen.ShaderBuiltins")
```

Comment:
Can't `nameof` be used here?

Reply:
`nameof` probably could / should be used in a lot more places, but it's kind of hit-or-miss in the current code. I'm fine with leaving it like this for now, although we should do a cleanup sweep sometime.